### ISSUE TITLE
[deprecated] Use of deprecated api in `lsp_extensions/workspace/diagnostic.lua`

### ISSUE THREAD

**Issue by kunzaatko** [tjdevries/lsp_extensions.nvim#33]
`vim.lsp.diagnostic.get`  is now being deprecated in favour of `vim.diagnostic.get`. There are more deprecations to be seen in `:h deprecated`.

**Comment by kunzaatko** [comment]
You can assign me to this issue however, I will have time only in about 2 weeks.

**Comment by kunzaatko** [comment]
akinsho/bufferline.nvim#272

**Comment by tjdevries** [comment]
I think perhaps it's better to not even have this anymore. You can just use `vim.diagnostic.get` since that already does workspaces.

I think I should just remove it completely, what do you think?

**Comment by kunzaatko** [comment]
Yeah, I think this would actually be a good solution. Maybe there should be a notice in the README or just a deprecation warning for a while, for the people that are using this still?

**Comment by kunzaatko** [comment]
In the case of a deprecation warning, which is the best solution in my opinion, the section of the README, should be removed in the meantime.

**Comment by j-hui** [comment]
@kunzaatko did you get a chance to work on this?

It actually took me weeks to get around to figuring out where this deprecation warning was coming from; the actual `vim.lsp.diagnostic.get_count is deprecated. See :h deprecated` warning doesn't exactly tell me where it's coming from so I had to do a binary search through my plugins...

**Comment by kunzaatko** [comment]
Hi! Sorry for the delay. I did not work on this but as TJ said, there is no reason in using this anymore, if I am not mistaken. All of the features that were provided by the plugin are now in neovim itself. Am I correct?
